src/components/Products/Products.jsx

```diff
@@ -1,9 +1,53 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { Card } from '../Card/Card.jsx'
+import s from './Products.module.css'
 
 export const Products = () => {
-    return <div>
-        <p>
-            Esses products vão dar trabalho...
-        </p>
+
+    const [page, atPage] = useState(1)
+    const [product, setProduct] = useState([])
```

Comment:
Como é um array de produtos, acho que no plural seria melhor
```suggestion
    const [products, setProducts] = useState([])
```